src/_includes/hero.html

```diff
@@ -11,7 +11,7 @@ <h1 class="usa-hero__heading">
         Some short explanatory text. You don’t need more than a couple of
         sentences.
       </p>
-      <a class="dgc-button usa-button" href="javascript:void(0)">Call to action</a>
+      <a class="usa-button" href="javascript:void(0)">Call to action</a>
```

Comment:
We'll need the `dgc-button` and `dgc-button--callout` class for the exercise. Doesn't seem big enough to have users add it themselves. 

Do you think we should have them modify?

Reply:
Currently we are instructing users to add it themselves

> 1. Go to hero component ./src/_includes/hero.html.
>2.  Add the following classes: dgc-button dgc-button--callout.

https://github.com/mejiaj/govcon-sandbox/blob/04-beyond/README.md#extending-button

I only made the change to have it match instructions but we can leave if you think that's better!

Reply:
Good point, thanks.